src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -191,6 +196,53 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
             }
         );
 
+        // update URL parameters according to sort direction UI events
+        // and DO NOT trigger a page refresh
+        this.urlParamsReaction = reaction(
+            ()=>[
+                this.sortMode,
+                this.sortByMutationType,
+                this.sortByDrivers
+            ],
+            ()=>{
+                const newParams = Object.assign({}, getBrowserWindow().globalStores.routing.location.query);
+                if (this.sortMode) {
+                    const mode = this.sortMode.type;
+                    let sortby_param = "";
+                    switch (mode) {
+                        case "alphabetical":               // sort by sample or patient id (a.k.a. alphabetical)
+                            sortby_param = "case_id";
```

Comment:
let's use an enum for the possible values of sortby_param instead of strings

Reply:
Updated. A string valued enum has been added replacing hard coded strings in the code:
```
export enum SortByUrlParamValue {
    CASE_ID = "case_id",
    CASE_LIST = "case_list",
    DATA = "data",
    DATA_MUTATION = "data_mut",
    DATA_DRIVER = "data_dri",
    DATA_MUTATION_DRIVER = "data_mut_dri",
    NONE = ""
}
```